lib/facility_factory.rb

```diff
@@ -0,0 +1,89 @@
+class FacilityInfo
```

Comment:
Remember our convention, the name of the class needs to match the file name. So this should be `class FacilityFactory`